ansible/roles/test/tasks/config_no_port_channels.yml

```diff
@@ -0,0 +1,85 @@
+- debug: msg="Configuration Test for topologies with No Port Channels"
+
+- name: gather interface facts
+  interface_facts:
+
+- name: set initial variables
+  set_fact:
+    active_eth_iface_list: [] 
+    test_ipv4_address: 10.200.4.5/31
```

Comment:
is this range portable? for example when LinkedIn tests on their side will it be the correct range?